client/omnichannel/directory/chats/contextualBar/ChatInfo.js

```diff
@@ -153,6 +156,14 @@ export function ChatInfo({ id, route }) {
 					<Label>{t('Closed_At')}</Label>
 					<Info>{formatDateAndTime(closedAt)}</Info>
 				</>}
+				{servedBy?.ts && <>
+					<Label>{t('Taken_At')}</Label>
+					<Info>{formatDateAndTime(servedBy.ts)}</Info>
+				</>}
+				{metrics?.response?.avg && <>
+					<Label>{t('Avg_response_time')}</Label>
+					<Info>{formatDuration(metrics.response.avg)}</Info>
```

Comment:
When the end-users creates a new room but the agent has not replied yet, I get this:

**The average of response time is empty:**

<img width="205" alt="Screen Shot 2021-03-31 at 12 05 54 PM" src="https://user-images.githubusercontent.com/59577424/113167158-d4629400-9219-11eb-939d-1fb3b7a02575.png">


**But there is a value store in the property(To be honest, I don't know why since the agent hasn't replied yet)**

<img width="452" alt="Screen Shot 2021-03-31 at 12 06 03 PM" src="https://user-images.githubusercontent.com/59577424/113167151-d298d080-9219-11eb-8b8f-ed435315f84d.png">